pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -4782,6 +4782,21 @@ typedef struct pjsua_acc_config
      */
     pj_bool_t        use_shared_auth;
 
+    /**
+    * Specify how to automatically process incoming SIP MESSAGE requests.
+    *
+    * When set to PJ_TRUE (automatic processing), incoming MESSAGE requests
+    * will be responded to immediately with SIP 200 OK (legacy behavior).
+    *
+    * When set to PJ_FALSE (manual processing), incoming MESSAGE requests
+    * will create UAS transactions and defer response sending. Applications
+    * must use pjsua_acc_send_response() to send responses manually.
```

Comment:
How about also adding a brief hint about the required steps for the app in the docs, for example:

> app must clone the request data (using ...) and obtain the transaction instance (using ...) from the `on_pager()/on_pager2()` callback, which is then used to send the response using `pjsua_acc_send_response()`.